src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer.Xaml/XamlLspServiceProvider.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Xaml;
+
+[Export(typeof(XamlLspServiceProvider)), Shared]
+internal class XamlLspServiceProvider : AbstractLspServiceProvider
+{
+    [ImportingConstructor]
+    [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+    public XamlLspServiceProvider(
+        [ImportMany(StringConstants.XamlLspLanguagesContract)] IEnumerable<Lazy<ILspService, LspServiceMetadataView>> lspServices,
```

Comment:
We may need to adjust the importing of the main service provider to include XAML handlers.  But we don't want an entirely separate set of XAML services (we want to share the workspace manager for example

Reply:
I just verified it's not needed at all.

Reply:
Gone